update outdated ansible deployment recommendations (#2837)
pingcap/docs: faq/tidb-faq.md

#### Deploy TiDB offline using TiDB Ansible

> **Warning:**
>

> > TiDB Ansible deployment method is deprecated since TiDB v4.0. [Use TiUP to deploy TiDB](/production-offline-deployment-using-tiup.md) instead.

Author: ```suggestion
> It is not recommended to deploy TiDB using TiDB Ansible since TiDB v4.0. [Use TiUP to deploy TiDB](/production-offline-deployment-using-tiup.md) instead.
```
